Automattic/jetpack
# Forms: Fix previews on Forms package

Fixes #29291

## Proposed changes:

* Fixes Forms previews on the Forms package

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No
## Testing instructions:

* Enable the Forms package by adding the following filter to the constructor of the `class.jetpack.php` inside the Jetpack plugin

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' ); 
```

* Create a post and add a Form block
* Check all the available block previews (Block Library, Block Style Variations, and Patterns Modal) work as expected
